staging/generic-ingress/templates/ingresses.yaml

```diff
@@ -0,0 +1,21 @@
+{{- range $index, $ingress := .Values.ingresses }}
+{{- $fullName := include "generic-ingress.fullname" $ }}
+---
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: {{ $ingress.name }}
```

Comment:
```suggestion
  name: {{ $ingressFullName }}
```

Reply:
I think we should be able to provide name per ingress so I am going to remove `ingressFullName` and keep `$ingress.name` value.

Reply:
This is a name per ingress, derived from each ingress.

Reply:
From https://github.com/mesosphere/charts/pull/481/files#r385750600

Reply:
Or alternatively just use this here:
```suggestion
  name: {{ include "generic-ingress.fullname" $ingress }}
```

Reply:
But if we have this non-unique default it can result in hard to debug result for no good reason. I think that is fair to ask for ingress object name from.

Reply:
Oh sorry i thought this was a parametrized template :man_facepalming: Ignore me.